test/cli/autocorrect-kwsplat/test.rb

```diff
@@ -1,29 +0,0 @@
-# typed: strict
-extend T::Sig
-
-sig {params(pos: Integer, x: Integer).void}
-def requires_x(*pos, x:)
-end
-
-sig {params(pos: Integer, x: String).void}
-def optional_x(*pos, x: '')
-end
-
-opts = T::Hash[Symbol, Integer].new
-string_keys = T::Hash[String, Integer].new
-nilble_opts = T::Hash[Symbol, T.nilable(Integer)].new
-x = T.let(:x, Symbol)
-
-requires_x(0, 1, 2, opts)
```

Comment:
This is the most common use case. It's also a valid code in 2.7, but not in 3.0.

With `--experimental-ruby3-keyword-args` it be auto-corrected like this:

```ruby
requires_x(0, 1, 2, **opts)
```

With the legacy handling like this

```ruby
requires_x(0, 1, 2, **T.unsafe(opts))
```

Both auto-corrects provide valid code, but the legacy solution adds unnecessary `T.unsafe`